src/components/organisms/RadioModal/RadioModal.tsx

```diff
@@ -1,20 +1,34 @@
-import React from "react";
+import React, { useCallback } from "react";
+import { Button } from "react-bootstrap";
 import { PLAYA_VENUE_NAME } from "settings";
 import "./RadioModal.scss";
```

Comment:
Would be good to remove playa from radio entirely:

```suggestion
import "./RadioModal.scss";
```